docs/Node Operators/configuration-variables.md

```diff
@@ -408,11 +408,13 @@ By default, Chainlink nodes write log data to `$ROOT/log.jsonl`. The log directo
 The `LOG_LEVEL` environment variable determines both what is printed on the screen and what is written to the log file.
 
 The available options are:
+- `"trace"`: Used only in development, off by default and not accessible on production builds.
 - `"debug"`: Useful for forensic debugging of issues.
 - `"info"`: High level informational messages.
-- `"warn"`: Something unexpected happened that might need non-urgent action. Node operators should check these once in a while to see whether anything stands out (e.g. deprecation warnings).
-- `"error"`: Something bad happened. Node operators might need to take urgent action based on this error.
-- `"panic"`: Something very bad happened. Node operators should take immediate action to fix this.
+- `"warn"`: A mild error occurred that might require non-urgent action. Check these warnings semi-regularly to see if any of them require attention. These warnings usually happen due to factors outside of the control of the node operator. Examples: Unexpected responded from a remote API or inaccurate networking errors.
+- `"error"`: An error occurred that might have put the node in a partially broken state. Node operators might need to take action to remedy this error. Examples: Deprecation warnings or incorrectly configured settings that cause a job to fail.
+- `"crit"`: A critical error occurred. The node might be completely stopped and non-responsive. Node operators should take immediate action to fix these errors. Examples: The node was not able to boot because a network socket could not be opened or the database became inaccessible.
```

Comment:
```suggestion
- `"crit"`: A critical error occurred. The node might be unable to function. Node operators should take immediate action to fix these errors. Examples: The node was not able to boot because a network socket could not be opened or the database became inaccessible.
```

Reply:
> Examples: The node was not able to boot because a network socket could not be opened or the database became inaccessible.
Are these good examples? They sound like fatal problems to me.

Reply:
There definitely are cases where we cannot open a socket but the node overall might still function. 

IIUC fatal kills the node. But from within libocr we likely shouldn't kill the node, it might also be doing non OCR things.

Reply:
Is a node in this partial state considered 'booted', or 'not able to boot'? From a developer perspective I was thinking that 'not able to boot' literally meant a blocking error during startup which is effectively fatal, and that this example/language should be reserved for that description. I think that generally we should try to document the _least severe_ examples at each level to illustrate the boundary most clearly.